### ISSUE TITLE
Homework upload

### ISSUE THREAD

**Issue by valentin4ahif** [rstropek-teaching/turmrechnen-quiz-4ahif-valentin4ahif#1]
This is the link to my homework which I made with StackBlitz: https://turmrechnenquiz.stackblitz.io

**Comment by rstropek** [comment]
Nearly. Your solution does not implement the specification in a few points:

* *Calculate* button should be disabled, not hidden
* Result should be a properly formatted table, not a list

Fix that and your extra point will be secured.

Greetings,
Rainer Stropek.

**Comment by valentin4ahif** [comment]
I corrected the two bulletpoints you told me to fix.
I added an Interface representing one line of a tower, changed the tower itself from a list to a table and changed the calculation algorithm, so that i am using the interface. Also, i put each element of the interface in another <td> Tag so that the operation symbols are in the same column.
The link to my homework is still the same as before.

Greetings,
Valentin Schützenberger, 4AHIF
